docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -23,7 +23,7 @@ Actors in the onboarding scenario:
 
 To enable above precondition and onboarding scenario in MVD, the following steps need to be implemented in the MVD infrastructure: 
 
-1. The MVD deployment workflow will generate the private and public keys for GAIA-X Authority.
+1. The MVD deployment workflow will generate the private and public keys for GAIA-X Authority. The public keys will be in [X.509 certificate](https://en.wikipedia.org/wiki/X.509) format.
```

Comment:
Actually JWK per RFC 7517 per did:web spec, we don't use X.509 at the time

Reply:
Thanks for the confirmation :)